pepper-apis/src/main/java/org/broadinstitute/ddp/db/dao/PicklistQuestionDao.java

```diff
@@ -499,23 +499,21 @@ default GroupAndOptionDtos findOrderedGroupAndOptionDtos(long questionId, long t
                     return result;
                 });
 
-        dtosMap.values().forEach(dto -> {
-            dto.getUngroupedOptions().forEach(parentDto -> {
-                if (nestedOptMap.containsKey(parentDto.getId())) {
-                    parentDto.getNestedOptions().addAll(nestedOptMap.get(parentDto.getId()));
-                }
-            });
-        });
+        dtosMap.values().forEach(dto -> dto.getUngroupedOptions().forEach(parentDto -> {
+            if (nestedOptMap.containsKey(parentDto.getId())) {
+                parentDto.getNestedOptions().addAll(nestedOptMap.get(parentDto.getId()));
+            }
+        }));
```

Comment:
This looks like an unnecessary change. Unless there is an issue such as a linter failing, let's revert this as the changes make it more difficult to read.

Reply:
This change was just for clearing up all the warnings. I changed the shape back to make it looks like before

Reply:
This change was just for clearing up all the warnings. I changed the shape back to make it looks like before